bibtasklets/bst_hepdata.py

```diff
@@ -74,11 +74,11 @@ def __iter__(self):
             results = requests.get("https://hepdata.net/search/", params=dict(size=chunk_size, page=page), headers={'Accept': 'application/json'}).json()['results']
             if not results:
                 break
-            for i, result in enumerate(results):
+            for result in results:
                 result = json_unicode_to_utf8(result)
                 paper_title = result['title']
                 inspire_id = result['inspire_id']
-                for data in result['data']:
+                for i, data in enumerate(result['data']):
```

Comment:
if you use start value 1 for enumerate `enumerate(result['data'], 1)`
you can use i directly and don't have to do `i+1` below